guillotina_amqp/job.py

```diff
@@ -61,7 +60,10 @@ def login_user(request, user_data):
     request.security.invalidate_cache()
     request._cache_groups = {}
     if user_data.get('Authorization'):
+        # leave in for b/w compat, remove at later date
         request.headers['Authorization'] = user_data['Authorization']
+    for name, value in (user_data.get('headers') or {}).items():
```

Comment:
won't this be faster? `user_data.get('headers', {}).items()`

Reply:
I always prefer the `.get(<something>) or default` so just in case there is ever a chance that `None` was a value in the structure...  Maybe it's not going to happen here but if you need something not-`None`, this is the most safe pattern to use.